src/style.css

```
@@ -0,0 +1,128 @@
+
+@font-face {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir", sans-serif;
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.bold-text {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.form-big {
+  position: relative;
+}
+
+.image-big {
+  margin-top: 11px;
+  position: absolute ;
+  width: 19px;
+  height: 19px;
+  padding: 26px;
+}
+
+.search-input-big {
+  margin-top: 12px;
+  width: 100%;
+  height: 66px;
+
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search-input-big:hover {
+  width: 100%;
+  height: 66px;
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search-input-big:focus {
+  width: 100%;
+  height: 66px;
+  border-radius: 0;
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search-input-big::placeholder,
+.search-input-big:hover::placeholder,
+.search-input-big:focus::placeholder {
+  padding: 62px;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+}
+
+.form-small {
+  position: relative;
+}
+
+.image-small {
+  margin-top: 22px;
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  padding: 13px;
```

you need to position this icon using top and left rules
```suggestion
  position: absolute;
  width: 11px;
  height: 11px;
```